src/lightning_lite/connector.py

```diff
@@ -63,8 +63,7 @@
 
 
 class _Connector:
-    """The Connector parses several Lite arguments and instantiates the Strategy including other components such as
-    the Accelerator and Precision plugins.
+    """The Connector parses several Lite arguments and instantiates the Strategy including it's owned components.
```

Comment:
```suggestion
    """The Connector parses several Lite arguments and instantiates the Strategy including its owned components.
```
What was wrong about the previous description?
To be clear, the precision class still formally remains a plugin, the same way the cluster environment is a plugin.

Reply:
I always prefer avoiding describing the code in docstrings, as they often go stale otherwise.